tmdrv_devices/thrustmaster_tmx.py

```diff
@@ -0,0 +1,9 @@
+name = 'Thrustmaster TMX'
+idVendor = 0x044f
+idProduct = [ 0xb65d, 0xb67e]
```

Comment:
Comparing this to the capture data that you provided, it looks like this might not be entirely correct. It looks to me like this should be:

```python
idProduct = [0xb67e, 0xb65d, 0xb67f]
```

In the `Thrustmaster_TMX_2017_07_22(Replug Only).pcap` capture file that you provided, packets numbered 466, 570, and 972 appear to indicate that these are the correct product IDs.